server/app.py

```diff
@@ -67,11 +76,12 @@ class Stop:
 class Cache:
   stops: typing.List[Stop]
   updated_at: str
-
+  
 PREDICTIONS_URL = 'https://api.511.org/transit/StopMonitoring'
 cache = Cache([], '')
 def update_cache():
     new_stops = []
+    cache.updated_at = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
how about instead we keep the `cache.updated_at` line where it is, that way if there is some error during updating, we dont accidentally say the cache was updated at a certain time when it wasnt


instead on  this line we can define a variable called `now`, and then assign `cache.updated_at` to `now` at the bottom

Reply:
resolved